python/ray/serve/drivers.py

```diff
@@ -174,3 +180,79 @@ async def get_intermediate_object_refs(self) -> Dict[str, Any]:
     async def get_dag_node_json(self) -> str:
         """Returns the json serialized root dag node"""
         return self.dags[self.MATCH_ALL_ROUTE_PREFIX].dag_node_json
+
+
+@PublicAPI(stability="alpha")
+class ServegRPCIngress:
```

Comment:
Nit: can we call this `gRPCIngress`? The `Serve` seems redundant considering this is in the `drivers.py` file in the `serve` directory.